cpp/cmake/Dependencies.cmake

```diff
@@ -39,7 +39,7 @@ else(DEFINED ENV{RAFT_PATH})
 
   ExternalProject_Add(raft
     GIT_REPOSITORY    https://github.com/rapidsai/raft.git
-    GIT_TAG           9dbf2c8a9134ce8135f7fe947ec523d874fcab6a 
+    GIT_TAG           16a3b8eda8b8c5948d2aad9dd9fcd09d7697e2f3
```

Comment:
This tag is outdated. Should bring to HEAD: 4a79adcb0c0e87964dcdc9b9122f242b5235b702

Reply:
Fixed.